js/classes/entities/foreground/crosshair.js

```diff
@@ -15,8 +15,7 @@ class Crosshair {
 
     update() {
         this.pos = GAME.mousePos;
-
-        if (GAME.mouseDown) {
+        if (GAME.user.aiming) {
```

Comment:
Thanks for doing the refactor here!

Reply:
Thanks for using the new naming convention